site/src/helpers/substitutions.ts

```diff
@@ -0,0 +1,52 @@
+import { ProgramRequirement } from '@peterportal/types';
+
+export const labelSubstitutions: Record<string, string> = {
+  Div: 'Division',
+  "Int'l": 'International',
+  US: 'United States',
+};
+
+export function expandAbbreviation(label: string): string {
+  let expandedLabel = label;
+
+  for (const [abbrev, substitution] of Object.entries(labelSubstitutions)) {
+    const regex = new RegExp(`\\b${abbrev}\\b`, 'gi');
+    expandedLabel = expandedLabel.replace(regex, substitution);
+  }
+
+  return expandedLabel;
+}
+
+export function findCommonLabelPrefix(labels: string[]): string {
+  if (labels.length === 0) return '';
+
+  const expandedLabels = labels.map(expandAbbreviation);
+
+  expandedLabels.sort();
+  /* remove all text after parentheses */
+  const first = expandedLabels[0].replace(/\s\(.*/g, '').split(' ');
+  const last = expandedLabels
+    .at(-1)!
+    .replace(/\s\(.*/g, '')
+    .split(' ');
+
+  let i = 0;
+  while (i < first.length && first[i] === last[i]) {
+    i++;
+  }
+  return first.slice(0, i).join(' ');
+}
+
+export function normalizeGERequirements(fetchedRequirements: ProgramRequirement[]): void {
```

Comment:
on the topic of naming functions, "inferLabelTitle" or something similar might be more descriptive, and may later be useful for non-GE requirements as well.